exec/cnex/lib/time_linux.c

```diff
@@ -1,6 +1,10 @@
 #define _POSIX_C_SOURCE 200809L
 #include <stdlib.h>
+#ifdef HAVE_CLOCK_GETTIME
```

Comment:
This doesn't work. On a system that does have clock_gettime (and cmake detects so), this compiler define still isn't set. Cmake variables are not the same thing as compiler defines.

Reply:
Added define in CMakeLists.txt